wire-desktop-beta/.SRCINFO

```diff
@@ -1,34 +1,24 @@
 pkgbase = wire-desktop-beta
-	pkgdesc = Modern, private messenger. Based on Electron.
+	pkgdesc = End-to-end encrypted messenger with file sharing, voice calls and video conferences
 	pkgver = 3.5.2881
-	pkgrel = 1
+	pkgrel = 2
 	url = https://wire.com/
 	arch = x86_64
-	arch = i686
 	license = GPL3
 	makedepends = cargo
 	makedepends = npm
 	makedepends = python2
 	makedepends = git
 	makedepends = yarn
-	depends = alsa-lib
-	depends = gconf
-	depends = gtk2
-	depends = libxss
-	depends = libxtst
-	depends = nss
+	depends = electron
 	depends = xdg-utils
-	optdepends = hunspell-en: for English spellcheck support
-	optdepends = noto-fonts-emoji: for colorful emoji made by Google
-	optdepends = ttf-emojione: for colorful emoji made by EmojiOne
+	optdepends = hunspell-en_US: for English spellcheck support
+	optdepends = emoji-font: colorful emoji
 	provides = wire-desktop
-	conflicts = wire-desktop-bin
-	conflicts = wire-desktop
```

Comment:
I made it so that `-beta` doesn't conflict with `wire-desktop`, so you can install them side-by-side and check how Wire looks on `dev` branch. They use different config directories (`~/.config/Wire` and `~/.config/WireInternal`), and I quickly tested that both can co-exist just fine.